projectDocs/community/expertsList.md

```diff
@@ -0,0 +1,25 @@
+# Community experts list
+
+This document outlines NV Access employees and community contributors.
+This can be used as a reference for people to contact or tag in regard to relevant issues and pull requests.
+
+## NV Access
+
+For optimal responses, please avoid tagging NV Access employees on unrelated tickets.
+General enquiries should be directed to <info@nvaccess.org>.
+
+| GitHub | email | topics |
+|---|---|---|
+| @gerald-hartig | <gerald@nvaccess.org> | Controversial product decisions |
+| @seanbudd | <sean@nvaccess.org> | Issue triage, technical decisions |
+| @SaschaCowley | <sascha@nvaccess.org> | Issue triage, technical decisions |
+| @michaelDCurran | <mick@nvaccess.org> | Advanced technical consulting |
+| @Qchristensen | <quentin@nvaccess.org> | User documentation, user support |
+| @jcsteh | <jamie@nvaccess.org> | Firefox bugs |
+
+## Community
+
+| GitHub | email | topics |
+|---|---|---|
+| @codeofdusk | <codeofdusk@gmail.com> | Windows Terminal and console, Microsoft UI Automation |
+| @nvdaes | <nrm1977@gmail.com> | Add-on store, braille |
```

Comment:
I'm fine with being tagged on LibreOffice-related issues/PRs.

Just as an idea: Would it make sense to switch the order, i.e. list the topic first, then the contact person? (My expectation would be that people looking into this document would usually first search for the topic, then look into the other columns.)

```suggestion
| @nvdaes | <nrm1977@gmail.com> | Add-on store, braille |
| @michaelweghorn | <m.weghorn@posteo.de> | LibreOffice |
```

Reply:
Thanks @michaelweghorn 

Given how fragments and specific our expertise is, we want to generate this list first before grouping by topics